angular.json

```diff
@@ -35,7 +35,7 @@
                 "output": "./assets"
               }
             ],
-            "styles": ["src/styles/main.scss"],
+            "styles": ["./node_modules/@angular/material/prebuilt-themes/deeppurple-amber.css", "src/styles/main.scss"],
```

Comment:
We don't need the whole pre-built purple theme, just the core. I fixed this long ago but pulled it out since it was unrelated to the PR I was working on, let me see if I can find it...

Reply:
Removed the css file from styles and disabled the theme check. ✅ 
[Docs](https://material.angular.io/components/core/api#MATERIAL_SANITY_CHECKS) | [Reference](https://github.com/angular/components/issues/16617)